observIQ/bindplane-op
# feat: Make the name parameter of the profile set command optional

### Proposed Change
If no `[name]` parameter is passed to the profile set command, use the current profile.

While I was in here, I also cleaned up some of the command descriptions and made the `--current` flag hidden since it isn't required.

```sh
Profile commands for managing BindPlane application configuration

Usage:
  bindplane profile [command]

Available Commands:
  create      Create a new profile
  current     Print the name of the current profile
  delete      Delete a profile
  get         Get details of a profile
  list        List all profiles
  set         Set details of a profile
  use         Set the current profile to the specified profile
```

To test, try

```sh
bindplane profile set --port 1
bindplane profile get  # confirm set to 1
bindplane profile set --port 3001

bindplane profile create tmp
bindplane profile set tmp --port 1
bindplane profile get tmp # confirm set to 1
bindplane profile delete tmp
```

##### Checklist
